test/unittests/commands/xmlupload/test_check_consistency_with_ontology.py

```diff
@@ -0,0 +1,314 @@
+from pathlib import Path
+
+import pytest
+from lxml import etree
+from pytest_unordered import unordered
+
+from dsp_tools.commands.xmlupload.check_consistency_with_ontology import (
+    _diagnose_all_classes,
+    _diagnose_all_properties,
+    _diagnose_class,
+    _diagnose_properties,
+    _find_problems_in_classes_and_properties,
+    _get_all_class_types_and_ids,
+    _get_all_classes_and_properties,
+    _get_all_property_names_and_resource_ids,
+    _get_prefix_and_prop_cls_identifier,
+)
+from dsp_tools.commands.xmlupload.models.ontology_diagnose_models import OntoCheckInformation, Ontology
+from dsp_tools.models.exceptions import BaseError
+
+# pylint: disable=missing-function-docstring,missing-class-docstring
+
+
+def test_get_all_classes_and_properties() -> None:
+    test_ele = etree.fromstring(
```

Comment:
General remark: For me, it feels more natural to abbreviate element with elem. But feel free to leave it as it is :-)